### DISCUSSION TITLE
default config not loading

### DISCUSSION THREAD

**Discussion by kaykhan** [polybar/polybar#2787] [Q&A] [answered]
Hi friends, im new to polybar. I am on ubuntu 22.04. Unfortunately i can't get the simple setup to load 

1. I have installed polybar uisng `sudo apt install polybar` https://github.com/polybar/polybar#installation

2. Following the first stps wiki https://github.com/polybar/polybar/wiki i should just be able to run `polybar` to have the "default" config load. However it does not load for me. 

![image](https://user-images.githubusercontent.com/5202527/185793353-699ec949-89d3-42cb-8c5b-e1410c1f6894.png)

The wiki suggests `/etc/polybar/config.ini` should exsit, but it does not exist.

**Comment by patrick96** [comment, marked as answer]
Ubuntu 22.04 only has polybar version 3.5.7, the newest version is 3.6.3. These particular features (default config and optional bar name) are only available on version 3.6 and up.

You can always [compile](https://github.com/polybar/polybar/wiki/Compiling) polybar yourself. Otherwise you will have to create your own config. There is a sample config included in the package, but it will in many cases not work out of the box, it's more about showing possible configuration options.

**Reply by kaykhan** [reply]
thank you, you  were right it was on version 3.5.7 i was able to compile polybar using the latest release.
